test/test_expecttest.py

```diff
@@ -96,6 +99,72 @@ def test_sample(self):
 ''')
 """)
 
+    def test_sample_lineno_at_start(self):
```

Comment:
nit: 
maybe specifically named previous test in line:49 as 
`test_sample_lineno_at_end(self)`